modules/dxp/apps/portal-reports-engine-console/portal-reports-engine-console-web/src/main/resources/META-INF/resources/tests/index.spec.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {render} from '@testing-library/react';
+import React from 'react';
+
+import {getDateValue} from '../js/index.js';
+
+describe('getDateValue', () => {
+	it('returns the date in the correct format', () => {
+		const parameterDate = new Date();
+
+		const day = parameterDate.getDay();
+		const month = parameterDate.getMonth();
+		const year = parameterDate.getYear();
+
+		render(<input id="parameterDateDay" value={day} />);
+		render(<input id="parameterDateMonth" value={month} />);
+		render(<input id="parameterDateYear" value={year} />);
+
+		const dateFormat = getDateValue('');
+		const date = dateFormat.split('-');
+
+		expect(Number(date[0])).toBe(year);
+		expect(Number(date[1])).toBe(month + 1);
+		expect(Number(date[2])).toBe(day);
```

Comment:
What do you think about creating the Date parameter using a specific date:

![image](https://github.com/user-attachments/assets/72b5cde6-91d8-45c7-8ea2-5af9a9f7c4b2)

and asserting exactly what the function returns? Something like this:

`expect(dateFormat).toBe('06-27-2025')`